es6-test/unit/geojson.js

```diff
@@ -1,25 +1,25 @@
 import chai from 'chai';
 import should from 'should';
-import * as es from 'event-stream';
-import {fixture} from '../fixture';
-import GeoJSON from '../../es6-lib/decoders/geojson';
-import {ArityChecker} from '../util';
+import es from 'event-stream';
+import { fixture } from '../fixture.js';
+import GeoJSON from '../../es6-lib/decoders/geojson.js';
+import { ArityChecker } from '../util.js';
+
 var expect = chai.expect;
 
 describe('geojson decoder', function() {
-
   it('will emit an error for malformed json', function(onDone) {
-    var count = 0;
+    const geoJsonInstance = new GeoJSON();
+    geoJsonInstance.prependListener('error', (error) => {});
```

Comment:
Changed test here--upgraded mocha so when unit test emits an error, the error is thrown and tests can't continue.